### PR TITLE
Assimilate l2-comms into a scripts folder

### PR DESCRIPTION
**Pull request by ajsantander**
This PR introduces a `./scripts` folder with some scripts we used during the Goerli Optimism trial deployment. Te idea is to have a place to put these scripts, and have a very easy way to add new ones or extend the existing.

### CODE REVIEW

File: scripts/status.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+	});
+
+	logItem(
+		'Synthetix.anySynthOrSNXRateIsInvalid:',
+		await Synthetix.anySynthOrSNXRateIsInvalid()
+	);
+	logItem('Synthetix.totalSupply', (await Synthetix.totalSupply()).toString() / 1e18);
+
+	/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ */
+	/* ~~~~ FixedSupplySchedule ~~~~ */
```
Comment (justinjmoses) [inline scripts/status.js:53]: Note: a lot of this is shared with SupplySchedule - perhaps we just use the SupplySchedule ABI and report on whatever both have in common?

File: scripts/status.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+			wallet,
+		});
+
+		logItem(
+			'FixedSupplySchedule.inflationStartDate',
+			new Date((await FixedSupplySchedule.inflationStartDate()).toString() * 1000)
+		);
+
+		const supply = bre.ethers.utils.formatEther(await FixedSupplySchedule.mintableSupply());
+		logItem('FixedSupplySchedule.mintableSupply', supply);
```
Comment (justinjmoses) [inline scripts/status.js:73]: Also we should split these out into a file per contract I reckon. I'd also love one for ExchangeRates that shows the rates (like the testnet script)

File: scripts/status.js (hunk 1 above)
Comment (ajsantander) [reply scripts/status.js]: Nice. Done.

File: scripts/status.js (hunk 2 above)
Comment (ajsantander) [reply scripts/status.js]: Improvements for the future? My goal in this PR is to reproduce what we had in l2-comms in this repo, in a way that is extensible and easy to use.

File: buidler.config.js [hunk 3]
```diff
@@ -11,6 +11,7 @@ require('@eth-optimism/ovm-toolchain/build/src/buidler-plugins/buidler-ovm-compi
 require('@eth-optimism/ovm-toolchain/build/src/buidler-plugins/buidler-ovm-node'); // add ability to start an OVM node
 
 usePlugin('@nomiclabs/buidler-truffle5'); // uses and exposes web3 via buidler-web3 plugin
+usePlugin('@nomiclabs/buidler-ethers');
```
Comment (justinjmoses) [inline buidler.config.js:14]: Is this still needed?

File: package.json [hunk 4]
```diff
@@ -70,9 +70,10 @@
 	},
 	"devDependencies": {
 		"@codechecks/client": "0.1.10",
-		"@eth-optimism/solc": "0.5.16-alpha.2",
 		"@eth-optimism/ovm-toolchain": "0.0.1-alpha.2",
+		"@eth-optimism/solc": "0.5.16-alpha.2",
 		"@nomiclabs/buidler": "1.3.7",
+		"@nomiclabs/buidler-ethers": "^2.0.0",
```
Comment (justinjmoses) [inline package.json:76]: and this?

File: scripts/airdrop.js [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+		gasPrice,
+		gasLimit,
+	};
+
+	for (const staker of data) {
+		const address = staker.address;
+
+		const collateral = parseEther(`${staker.collateral}`);
+
+		const l2Balance = await Synthetix.balanceOf(address);
```
Comment (justinjmoses) [inline scripts/airdrop.js:78]: we should also check make sure there have been no transfer events in some recent window of time.

File: scripts/status.js [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+		provider,
+	});
+
+	const logRate = async currency => {
+		const rate = await ExchangeRates.rateForCurrency(formatBytes32String('SNX'));
+		const updated = (await ExchangeRates.lastRateUpdateTimes(formatBytes32String('SNX'))) * 1000;
+		logItem(`${currency} rate:`, `${rate} (updated ${updated})`);
+	};
+
+	await logRate('SNX');
```
Comment (justinjmoses) [inline scripts/status.js:174]: could we also log all synths?

**Review by justinjmoses — CHANGES_REQUESTED** [review]
A couple of minor changes

File: buidler.config.js (hunk 3 above)
Comment (ajsantander) [reply buidler.config.js:14]: Good catch, thanks.

File: scripts/airdrop.js (hunk 5 above)
Comment (ajsantander) [reply scripts/airdrop.js:78]: Do you think this is still required, given that the script now demands that the system be suspended?

Comment (justinjmoses) [reply scripts/airdrop.js:78]: Ah well you can't transfer SNX at all if the system is suspended. 🤔

File: scripts/status.js [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+	const logRate = async currency => {
+		const rate = await ExchangeRates.rateForCurrency(formatBytes32String(currency));
+		const updated = (await ExchangeRates.lastRateUpdateTimes(formatBytes32String(currency))) * 1000;
+		logItem(`${currency} rate:`, `${formatEther(rate)} (${new Date(updated.toString() * 1000)})`);
+	};
+
+	await logRate('SNX');
+
+	const synthsPath = path.join(getPathToNetwork({ network, useOvm }), constants.SYNTHS_FILENAME);
+	const synths = JSON.parse(fs.readFileSync(synthsPath));
```
Comment (justinjmoses) [inline scripts/status.js:249]: Why not use `getSynths()` ?

Comment (ajsantander) [reply scripts/status.js:249]: Nize

**Review by justinjmoses — APPROVED** [review]
:shipit:

**Pull request merged**